src/Makefile.am

```diff
@@ -468,7 +468,8 @@ libprotoc_la_SOURCES =                                         \
   google/protobuf/compiler/csharp/csharp_source_generator_base.cc \
   google/protobuf/compiler/csharp/csharp_source_generator_base.h \
   google/protobuf/compiler/csharp/csharp_wrapper_field.cc      \
-  google/protobuf/compiler/csharp/csharp_wrapper_field.h
+  google/protobuf/compiler/csharp/csharp_wrapper_field.h       \
+  google/protobuf/compiler/csharp/csharp_options.h
```

Comment:
Sorry to be even more of a pain, but all the files are listed alphabetically - could you move this line appropriately?